app/gui2/src/composables/syncLocalStorage.ts

```diff
@@ -0,0 +1,142 @@
+import { useAbortScope } from '@/util/net'
+import { debouncedWatch, useLocalStorage } from '@vueuse/core'
```

Comment:
I didn't know there is something like `debounceWatch`. You may check if it's suitable to our `debouncedGetter` (my implementation uses timeouts).

Reply:
Yes, I think is actually a suitable and easy to understand replacement for manual timers in this case. I'm going to refactor it then.